### PR TITLE
Get data from Gobierto Datos

### PR DESCRIPTION
**Pull request by jorgeatgu**
Closes 

PopulateTools/issues#1128
PopulateTools/issues#1124
https://github.com/PopulateTools/issues/issues/1129
https://github.com/PopulateTools/issues/issues/1130

## :v: What does this PR do?

WIP!

## :mag: How should this be manually tested?

Staging

### CODE REVIEW

File: app/views/gobierto_budgets/pages/_map_metrics.html.erb [hunk 1]
```diff
@@ -7,6 +7,7 @@
     data-line-widget-series="means"
     data-line-widget-type="per_person"
     data-widget-data-url="<%= gobierto_budgets_api_global_data_total_budget_per_inhabitant_path(year: @year, format: :json) %>">
+  >
```
Comment (ferblape) [inline app/views/gobierto_budgets/pages/_map_metrics.html.erb:10]: Careful

**Comment by ferblape** [comment]
@furilo ready for review in http://presupuestos.gobify.net/mapas/2020

File: app/assets/javascripts/gobierto_budgets/maps.js [hunk 2]
```diff
@@ -202,67 +62,438 @@ $(document).on('turbolinks:load', function() {
...
+          viewState: viewState
+        });
+      }
+    });
+
+
+    function onClick(info, event) {
+      var municipality = info.object.properties.name
+      var replaced = municipality.replace(/ /g, '-');
+      window.location.href = "/places/" + accentsTidy(replaced) + "/" + year;
```
Comment (ferblape) [inline app/assets/javascripts/gobierto_budgets/maps.js:125]: This function is broken, i.e http://presupuestos.gobify.net/places/gallues/galoze/2020

You should use the `slug` attribute from municipalities table, which already removes accents and so on.

**Comment by ferblape** [comment]
@jorgeatgu ping me when you are back to this project, because there's a special case which is not solved in this PR, which is the map for catalonia: http://pressupostosmunicipals.transparenciacatalunya.cat/mapas/2020, in staging seems broken: http://gencat-presupuestos.gobify.net/mapas/2020

**Comment by jorgeatgu** [comment]
Add a new condition to filter municipalities data if `window.placesScope` contains ine codes.

Set attributes latitude, longitude, and zoom from `window.Settings`, carto zoom is a bit different than the deck.gl zoom.

Screen Recording 2020-11-04 at 12 09 33 2020-11-04 12_28_30

**Pull request merged**